social-rs/server/src/post.rs

```diff
@@ -7,7 +7,7 @@ use uuid::Uuid;
 #[collection(posts)]
 pub struct Post {
     pub text: String,
-    #[owned_by(User, user_id)]
+    #[owned_by(User)]
```

Comment:
how do we want to enforce what the field this corresponds to in the User? Should we be implementing it such that the User struct itself is annotated with something to represent the "primary key"? @swetabhch would love your thoughts on this too

Reply:
Agree keeping track of UUID is ideal — is it reasonable to just split on the string here? Reason I pulled it was because it was making the inner logic a bit gnarlier, that's maybe the best current solution?